src/style.css

```
@@ -1 +1,100 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap");
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 450;
+  
+}
+
+body {
+
+  margin: 0;
+
+}
+
+ul {
+
+  margin: 0;
+  
+}
```

why not combine them?

```suggestion
body,
ul {
  margin: 0;
}
```